submissions/tosinIshola/HTML/Home.html

```diff
@@ -0,0 +1,44 @@
+<html lang="en">
```

Comment:
Which page is this? Where are ask to create a home page? If no, kindly remove it